source/administration/config-settings.rst

```diff
@@ -3851,7 +3851,7 @@ Set whether or not this Mattermost server will schedule tasks that will be compl
 
 When running Mattermost on a single machine, this setting should always be enabled.
 
-When running Mattermost in High Availablity mode, this setting should always be enabled. In a High Availability cluster, exactly one of the servers will be designated as the Scheduler at a time to ensure that duplicate tasks aren't created.
+When running Mattermost in High Availablity mode, this setting should always be disabled. In a High Availability cluster, exactly one of the servers will be designated as the Scheduler at a time to ensure that duplicate tasks aren't created.
```

Comment:
OK seeing the old docs I'm now worried something has gone badly wrong here. I'll investigate...

Reply:
@grundleborg This was recently added, so it might have also been a misunderstanding. But @hmhealey might know more about that.

Reply:
Last we had discussed this, my understanding was that all servers would have `RunScheduler` enabled and the cluster leader would assign one of them to be the scheduler

Reply:
@grundleborg Any updates on this one?

Reply:
I just looked back at my chat history, and when Corey and I were discussing it, we thought that it had to be on for all servers and then the leader node would take care of figuring out which server was actually the scheduler. Perhaps we were wrong about that? It was in a DM so I can send you the chat transcript

Reply:
I've had a look through the code and I have no idea any more. I think Corey implemented this, so it might be worth us asking him what the intended behaviour was and how it worked.

Reply:
Discussing off-github with Corey and Harrison.

Reply:
Update: code fix pending.

Reply:
@jasonblais ok so now the code fix has gone in, the options are:
1. Run a single jobserver *without* the `--noschedule` flag. That jobserver will be solely responsible for scheduling recurring jobs.
2. Run all jobservers with `--noschedule` flag, and then set this config to `true` for *all* app servers in the cluster. The cluster leader at any given time will then be responsible for scheduling recurring jobs.

I think 2 is the recommended option in a properly configured HA cluster, as it provides redundancy in job scheduling capability, which would not be present if the only jobserver with scheduling responsibility were to fail.